plugins/org.eclipse.fordiac.ide.ant/src_ant/org/eclipse/fordiac/ide/ant/ant/Import4diacProject.java

```diff
@@ -55,6 +55,10 @@ public void execute() throws BuildException {
 			workspace.setDescription(desc);
 			Job.getJobManager().cancel(null);
 
+			// ensure SystemManager is loaded
+			SystemManager.INSTANCE.removeFordiacChangeListener();
+			SystemManager.INSTANCE.addFordiacChangeListener();
```

Comment:
```suggestion
			SystemManager.INSTANCE.hashCode(); // ensure SystemManager is instantiated
```

I think this should also ensure that the instance exists without the side-effect of switching the change listener off&on again

Also: based on the comments in SystemManager this change might make the call to `TypeLibraryManager.INSTANCE.getTypeLibrary(project).reload();` on line 71 redundant as instatiation of the TypeLib is also ensured by SystemManager; I'd leave that in for now and test later

Reply:
I would prefere:

`@SuppressWarnings("unused")
finalSystemManager mgr = SystemManager.INSTANCE; `

As we are doing it in in the activator of the org.eclipse.fordiac.ide plugin.  There it is sufficient.

Reply:
replaced the line with this